src/pages/SignUp.jsx

```diff
@@ -15,68 +19,71 @@ export default function SignUp() {
       },
-      onError: () => {
-        toast.error("Registration Failed");
+      onError: (error) => {
+        toast.error("Registration Failed: " + error.message);
       },
     }
   );
 
+  const onSubmit = async (data) => {
+    await signUp({
+      ...data,
+      role: ROLES.EMPLOYER,
+    });
+  };
+
   return (
     <div className="flex flex-col justify-center items-center">
       <div className="card max-w-md">
         <div className="card-body">
           <p className="mb-5 text-2xl">Register With RefugeeOne Job Search Portal</p>
-          <form onSubmit={handleSubmit(signUp)}>
+          {/* @tianchi @neha todo: may add an explantion for the sign up here */}
+          <p>@tianchi @neha We could add a explanation here</p>
+          <form onSubmit={handleSubmit(onSubmit)}>
             <label className="label" htmlFor="name">
               <span className="label-text">Full Name</span>
             </label>
             <input
               {...register("name", { required: true })}
               type="text"
-              className="input w-full max-w-xs input-bordered"
+              className="input w-full input-bordered"
             />
             <label className="label" htmlFor="email">
               <span className="label-text">Email</span>
             </label>
             <input
               {...register("email", { required: true })}
               type="email"
-              className="input w-full max-w-xs input-bordered mb-4"
+              className="input w-full input-bordered mb-4"
             />
             <label className="label" htmlFor="compnay">
               <span className="label-text">Company Name</span>
             </label>
             <input
               {...register("company")}
               type="text"
-              className="input w-full max-w-xs input-bordered mb-4"
+              className="input w-full input-bordered mb-4"
             />
             <label className="label" htmlFor="phone">
               <span className="label-text">Phone Number</span>
             </label>
             <input
               {...register("phone", { required: true })}
               type="number"
-              className="input w-full max-w-xs input-bordered mb-4"
+              className="input w-full input-bordered mb-4"
             />
             <label className="label" htmlFor="password">
               <span className="label-text">Password</span>
             </label>
             <input
               {...register("password", { required: true })}
               type="password"
-              className="input w-full max-w-xs input-bordered mb-4"
-            />
-            <label className="label" htmlFor="role">
-              <span className="label-text">Role</span>
-            </label>
-            <input
-              {...register("role", { value: "employer" })}
-              type="text"
-              className="input w-full max-w-xs input-bordered mb-4"
```

Comment:
Why are we removing this disabled role field? I guess we can leave this role field in the disabled form so that if someone (refugee or admin) by mistake clicks on the "register as employer" they will know that this form is specific for employer role and not for any other role